app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -1,7 +1,7 @@
-<%# OVERRIDE Hyrax v5.0.0rc2 Adjust permission checks and add analytics button %>
+<%# OVERRIDE Hyrax v5.0.0rc2 Adjust permission checks and add analytics button and add ENV to control turbolinks %>
 <div class="show-actions">
   <% if presenter.editor? %>
-      <%= link_to t('.edit'), edit_polymorphic_path([main_app, presenter]), class: 'btn btn-secondary' %>
+      <%= link_to t('.edit'), edit_polymorphic_path([main_app, presenter]), class: 'btn btn-secondary', data: { turbolinks: ENV.fetch('TURBOLINKS_FOR_VALKYRIE', 'true') } %>
```

Comment:
I think all of these `ENV.fetch` lines need to be cast as a boolean ([example](https://github.com/scientist-softserv/palni-palci/blob/b880891b053becbe73d90bbddc8e4c8baae87afb/app/controllers/application_controller.rb#L121)). I _think_ ENV vars are always strings, so `"false"` will actually be truthy

Reply:
Hmmm, I'm not trying to check for truthy or falsey, I'm looking to set it to either string "true" or "false" so it would evaluate to `data: { turbolinks: 'true' }` or `data: { turbolinks: 'false' }`